.github/workflows/DeployMain.yml

```diff
@@ -1,5 +1,5 @@
 # This is a workflow based on an example provided by GitHub and on rossjrw/pr-preview-action
-name: Deploy main Playground
+name: Deploy main website
```

Comment:
It is fine to rename it, but please do rename the name of the preview workflow as well!